src/grokcore/component/tests/test_grok.py

```diff
@@ -18,7 +18,12 @@ def cleanUpZope(test):
     # exceptions appear in traceback printouts.
     (re.compile(r"ConfigurationExecutionError: <class '([\w.]+)'>:"),
                 r'ConfigurationExecutionError: \1:'),
-    ])
+    # unicode objects doese not more exist in python3.
```

Comment:
s/doese/do/

(also, I think 'unicode object representation changed in Python 3' would be a better phrasing)